backstop/tests/menu.html

```diff
@@ -617,72 +617,87 @@ <h3>Header</h3>
     <section id="demo-header">
 
       <header class="iui-page-header">
-        <div class="iui-left">
+        <div class="iui-page-header-left">
           <!-- Application Button -->
           <div
-            class="iui-header-logo"
+            class="iui-header-brand"
             role="button"
             aria-label="App name"
             tabindex="0"
           >
             <svg-imodel-hollow
-              class="iui-header-button-icon"
+              class="iui-header-brand-icon"
               aria-hidden="true"
             ></svg-imodel-hollow>
-            <span class="iui-label">App name</span>
+            <span class="iui-header-brand-label">App name</span>
           </div>
 
-          <div class="iui-divider"></div>
+          <div class="iui-page-header-divider"></div>
 
-          <nav aria-label="breadcrumbs">
-            <!-- Project Button -->
-            <button
-              class="iui-button iui-borderless iui-large iui-header-button iui-header-dropdown-button"
-              type="button"
-              aria-label="Project picker"
-              aria-expanded="false"
-              aria-haspopup="true"
-              id="test-button-1"
-            >
-              <div class="iui-button-icon iui-header-button-icon demo-picture"></div>
-              <span class="iui-button-label">
-                <div>Project Alpha</div>
-                <div class="iui-description">0x0123456789</div>
-              </span>
-              <svg-caret-down-small
-                class="iui-button-icon"
-                aria-hidden="true"
-              ></svg-caret-down-small>
-            </button>
-
-            <svg-chevron-right
-              class="iui-chevron"
-              aria-hidden="true"
-            ></svg-chevron-right>
-
-            <!-- IModel Button -->
-            <button
-              class="iui-button iui-borderless iui-active iui-large iui-header-button iui-header-dropdown-button"
-              type="button"
-              aria-label="Model picker"
-              aria-expanded="true"
-              aria-haspopup="true"
-              aria-current="location"
-              id="demo-header-imodel-button"
-            >
-              <svg-model
-                class="iui-button-icon iui-header-button-icon"
-                aria-hidden="true"
-              ></svg-model>
-              <span class="iui-button-label">
-                <div>Model Beta</div>
-                <div class="iui-description">0x0123456789</div>
-              </span>
-              <svg-caret-down-small
-                class="iui-button-icon"
+          <nav
```

Comment:
There appears to be an overflow bug somewhere in the breadcrumbs.  I'm trying to figure out where the overflow needs to be applied & then figure out if the change belongs in `header.scss`, `breadcrumbs.scss`, or `@mixin iui-label-with-sublabel`.